css/layouts/search.scss

```diff
@@ -0,0 +1,64 @@
+// styling for search popover from header
+
+.header-right {
+    .unified-search__input-wrapper label {
+        display: none;
+    }
+
+    .unified-search__input-wrapper ~ .empty-content {
+        display: none;
+    }
+
+    .empty-content {
+        &__icon > .material-design-icon.magnify-icon {
+            display: none;
+        }
+    }
+
+    .header-menu {
+        &__carret {
+            display: none;
+        }
+    }
+}
+
+.unified-search__form {
+    & #unified-search__input {
+        border: 2px solid black;
+        border-radius: 5px;
+        padding-top: 1.5rem;
+        padding-bottom: 1.5rem;
+        padding-left: 1rem;
+    }
+
+    & .unified-search__form-reset.icon-close {
+        background-image: var(--icon-close-x-dark);
+        opacity: 1;
+        margin: 12px 3px;
+    }
+}
+
+.unified-search__result-icon.unified-search__result-icon--no-preview {
+    &.icon-folder {
```

Comment:
Please do not exchange a single folder icon in the result list. All non-preview icons must come from Mimetype mapping in Nextcloud - we have a support request for that pending.
(even if Jens mentioned this in the ticket...)

Reply:
Thank you for this insight, it seems that this change has been implemented in the MIME Type mapping (customapps/nmctheme/css/apps/media.scss) since Friday, so I have removed this from search.scss